application/core/Survey_Common_Action.php

```diff
@@ -850,23 +850,19 @@ function _surveybar($aData)
 
             $aData['gid'] = $gid; // = $this->input->post('gid');
 
-            if (Permission::model()->hasSurveyPermission($iSurveyID, 'surveycontent', 'read'))
-            {
+            if (Permission::model()->hasSurveyPermission($iSurveyID, 'surveycontent', 'read')) {
```

Comment:
https://manual.limesurvey.org/Coding_guidelines#Structures when it was updated ? Sorry o follow the old coding guideline since ..... the old coding guidelins of LimeSurvey (3 years old or less ? https://manual.limesurvey.org/index.php?title=Coding_guidelines&oldid=43745#Structures )